sql/connect/common/src/main/protobuf/spark/connect/pipelines.proto

```diff
@@ -149,6 +150,19 @@ message PipelineCommand {
       optional spark.connect.Relation relation = 1;
     }
 
+    // A standalone flow that writes to the target dataset with additional options.
+    message StandaloneFlowDetails {
```

Comment:
we can migrate `dp.create_flow` to use this later, when actually start supporting `once` for example.